subsys/net/openthread/settings_check/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# This script checks if NVS settings backend is used with nRF54L SoC target and prints
+# the warning that the recommended backend is ZMS due to the RRAM design.
+
+if (CONFIG_SOC_SERIES_NRF54LX AND CONFIG_SETTINGS_NVS)
```

Comment:
does not take into consideration if NVS is being placed on external flash

Reply:
Yes, because it does consider only OpenThread module use case, which uses only settings. This is not generic warning for all application use cases.

Reply:
it does not work though because if I use NVS on **external flash** with the Kconfig enabled for that, I wrongly get this giant warning when NVS isn't on the internal RRAM, it's on an external flash

Reply:
Ok, so do you mean settings placed on the external flash or NVS used for some custom use case on the external flash?